client/src/pages/transaction/tokens-create.tsx

```diff
@@ -0,0 +1,40 @@
+import { Code, Table, Tbody, Td, Tr } from "@liftedinit/ui";
+import { Link } from "react-router-dom";
+
+export function tokensCreate(data: any) {
+  const summary = data.argument.summary;
+  return {
+    Token: (
+      <Code as={Link} to={`/addresses/${data.argument.owner}`} color="brand.teal.500">
+        {summary.ticker}
+      </Code>
+    ),
+    Holders: (
+      <Table>
```

Comment:
How does it render when you have more than a single dest?

Reply:
It renders a table of each holder with the initial distribution. Would you prefer a list of holders and a single distribution value?

Reply:
Is it easy to show me both?

Reply:
Here's the data I'm receiving. Are there cases where the initial distribution will differ per address? 
```
{
  "summary":{
     "name":"Test",
     "ticker":"TST",
     "precision":9
  },
  "owner":"mahmcjf5yyodcpomqrcnqfkbpltt45oygniar7y32p7ltykqnd",
  "holders":{
     "mahmcjf5yyodcpomqrcnqfkbpltt45oygniar7y32p7ltykqnd":"10000000000000000"
  },
  "extendedInfo":{
     
  },
  "memo":[
     
  ]
}
```

Reply:
Yes. The initial distribution can be anything. E.g.,
```json
{
  "maaaaaaaaaaaaaaaaaaaaaaaaaaaaaa": 123,
  "mabbbbbbbbbbbbbbbbbbbbbbbbbbbbb": 456,
}
```

Reply:
Then I suggest we leave it as the table